.github/workflows/ci.yml

```diff
@@ -6,20 +6,51 @@ on:
 
 env:
   CONSUL_VERSION: 1.6.10
```

Comment:
No we are testing different version of consul, so I think that variable is not needed anymore

Reply:
Indeed, done